builtin/Cargo.toml

```diff
@@ -10,36 +10,36 @@ edition = "2021"
-fil_actor_paych = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
-fil_actor_power = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
-fil_actor_reward = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
-fil_actor_system = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
-fil_actor_verifreg = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
-fil_actors_runtime = { version = "10.0.0-alpha.1", git = "https://github.com/filecoin-project/builtin-actors", rev = "d2924bfd", features = [] }
+actors-v10 = { package = "fil_builtin_actors_bundle", version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps" }
+fil_actor_account = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_cron = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_datacap = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_init = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_market = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_miner = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_multisig = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_paych = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_power = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_reward = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_system = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actor_verifreg = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
+fil_actors_runtime = { version = "10.0.0", git = "https://github.com/filecoin-project/builtin-actors", branch = "alex/update-deps", features = [] }
```

Comment:
Reminder to update these back to master before merging

Reply:
TODO(@alexytsu): update after https://github.com/filecoin-project/builtin-actors/pull/1254 lands

Reply:
pinned to a commit with v10 actors still for now. can update to a 11.0.0-alpha commit (now or later) if that's better